### ISSUE TITLE
ChoiceGroup: cannot use arrows to move to other radio controls on Edge with Narrator

### ISSUE THREAD

**Issue by ssingh28** [OfficeDev/office-ui-fabric-react#5285]
On Edge with Narrator On it is not possible to navigate from one option to another of choice group. This behavior is also available on MS Fabric React page https://developer.microsoft.com/en-us/fabric#/components/choicegroup . Even here when the narrator is on it has to go through text on each individual buttons and righ and left cursor keys don't work..

**Comment by dzearing** [comment]
@kkjeer Can you please help investigate this? I'm unsure if this is a real issue or an Edge/Navigator issue.

**Comment by JasonGore** [comment]
ChoiceGroup navigation seems to be working fine with me when using Narrator and Edge. I can switch between options as expected. It does not go through text.

This seems to be a duplicate of #4938.

Are you sure [scan mode](https://support.microsoft.com/en-us/help/22809/windows-10-narrator-using-scan-mode) is not enabled?

**Comment by JasonGore** [comment]
I'm going to close this as a duplicate of #4938.

**Comment by ssingh28** [comment]
@dzearing @JasonGore @kkjeer : The issues I have reported are w.r.t to keyboard navigation on Edge with Narrator On. These issues may not be reproducible when you choose mouse control instead.
Hence please have a look into them again.

**Comment by JasonGore** [comment]
Please read the link I provided about scan made. This has nothing to do with mouse control.

Regardless, this bug is still a duplicate of #4938.
